packages/terra-alert/src/terra-dev-site/test/alert/AlertPopup.test.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from 'terra-button';
+import Alert from '../../../Alert';
+
+class AlertActionButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      actionButtonClickCount: 0,
+      showAlert: false,
+    };
+    this.actionFunc = this.actionFunc.bind(this);
+    this.popAlert = this.popAlert.bind(this);
+    this.alert = (
+      <Alert id="actionAlert" type="warning" action={<Button text="Action" onClick={this.actionFunc} />}>
+    This is a warning. It is configured with a custom Action button. Action button has been
+    clicked
+        {' '}
+        <span id="actionButtonClickCount">{this.state.actionButtonClickCount}</span>
+        {' '}
+    times.
+      </Alert>
+    );
+  }
+
+  actionFunc() {
+    this.setState(prevState => ({ actionButtonClickCount: prevState.actionButtonClickCount + 1 }));
+  }
+
+  popAlert() {
+    this.setState(prevState => ({ showAlert: !prevState.showAlert }));
+  }
+
+  render() {
+    return (
+      <div>
+        <Button text="Action" onClick={this.popAlert} />
```

Comment:
It might be helpful to add a simple description to this test to mention this is for functional validation for Jaws / Voice over since we arn't using it in wdio

Reply:
Added 8b24860